src/libraries/System.Private.CoreLib/src/System/IO/Strategies/FileStreamCompletionSource.Win32.cs

```diff
@@ -6,262 +6,244 @@
 using System.Runtime.InteropServices;
```

Comment:
@adamsitnik gave me the suggestion of keeping the filename shorter (remove the parent class). Either way is fine by me.

Reply:
This is one of the reasons I wish we would use nested types here.  You wouldn't need FileStream in the name at all, e.g. the type would just become "CompletionSource" because it would be within FileStream, we wouldn't need this weird System.IO.Strategies namespace (which ends up being internal only, looks like it's general but is specific to FileStream, etc.)

Reply:
I'll rename this type to `Net5CompatFileStreamStrategy.CompletionSource.Win32.cs`